src/bindings/c/src/ov_core.cpp

```diff
@@ -17,6 +17,24 @@ char* str_to_char_array(const std::string& str) {
     return char_array;
 }
 
+static char* last_err_msg = nullptr;
+static std::mutex last_msg_mutex;
+void dup_last_err_msg(const std::string msg) {
```

Comment:
May be it was meant `dump_last_err_msg`?

Reply:
dup means duplicate.